src/components/expanded-state/ContactProfileState.js

```diff
@@ -85,7 +84,10 @@ const ContactProfileState = ({ address, color, contact, ens, nickname }) => {
   const avatarUrl = profilesEnabled ? avatar?.imageUrl : undefined;
 
   const { result: dominantColor } = usePersistentDominantColorFromImage(
-    maybeSignUri(avatarUrl || '', { w: 200 }) || ''
+    avatarUrl,
+    {
+      signUrl: true,
```

Comment:
i would rather call `maybeSignUri` internally every time

it should detect that its already signed and do nothing if that is the case

Reply:
I changed it